Modules/CalcPerform.lua

```diff
@@ -423,6 +423,11 @@ local function doActorMisc(env, actor)
 			modDB:NewMod("SpellDodgeChance", "BASE", m_floor(15 * effect), "Elusive")
 			modDB:NewMod("MovementSpeed", "INC", m_floor(30 * effect), "Elusive")
 		end
+		if modDB:Flag(nil, "Blind") then
```

Comment:
Maintainers might feel different since this would be slightly different than the surrounding buffs/debuffs, but I believe it would be better to do this all in one if statement:

`if modDB:Flag(nil, "Blind") and not modDB:Flag(nil, "IgnoreBlindHitChance") then`

Reply:
I wrote the condition in one line initially but then decided to split it up. I think it's more readable if it follows the style of surrounding code and it's going to be easier to add light radius interaction if we ever figure it out.